shop-be/product-service/src/functions/dbInit/handler.ts

```diff
@@ -0,0 +1,34 @@
+import { dbConnectAndExecute } from '@libs/db';
+
+const createProductsTable = `
```

Comment:
Such things are usually called `migration scripts` and it can be good to have a dedicated folder and common runtime script for them.

Reply:
moved it to /migrations, but need some more time for investigations(